sdk/src/trace/span.h

```diff
@@ -45,6 +45,13 @@ class Span final : public opentelemetry::trace::Span
   void SetStatus(opentelemetry::trace::StatusCode code,
                  nostd::string_view description) noexcept override;
 
+#if OPENTELEMETRY_ABI_VERSION_NO >= 2
+  void AddLink(const opentelemetry::trace::SpanContext &target,
```

Comment:
nit: put both declaration to be above the `SetStatus`, to be consistent with the order in .cc file.

Reply:
Fixed.